src/components/newsBox/newsBox.module.css

```diff
@@ -11,7 +11,11 @@
 }
 
 .container:hover .title {
-  color: var(--secondary-color);
+  opacity: 0.6;
+}
+
+.container:hover {
+  border-color: rgba(211, 211, 211, 0.632);
```

Comment:
will it work for both dark/light themes?

Reply:
implemented changes